modules/dcache-vehicles/src/main/java/diskCacheV111/vehicles/JobInfo.java

```diff
@@ -4,55 +4,75 @@
 
 import static com.google.common.base.Preconditions.checkNotNull;
 
 public class JobInfo implements Serializable {
 
-    private static final SimpleDateFormat __format =
-            new SimpleDateFormat( "MM/dd-HH:mm:ss" ) ;
+    private final DateTimeFormatter __formatter = DateTimeFormatter.ofPattern("MM/dd-HH:mm:ss");
+
     private static final long serialVersionUID = 5209798222006083955L;
 
-   private final String _client;
-   private final long   _clientId;
-   private final long   _submitTime;
-   private final long   _startTime;
-   private String _status;
-   private final long   _jobId;
-
-    public JobInfo(long submitTime, long startTime, String status, int id, String clientName , long clientId ){
-      _submitTime = submitTime ;
-      _startTime  = startTime;
-      _status     = checkNotNull(status);
-      _jobId      = id;
-      _client   = clientName ;
-      _clientId = clientId ;
-   }
-   public String getClientName(){ return _client ; }
-   public long   getClientId(){ return _clientId ; }
-   public long   getStartTime(){ return _startTime ; }
-   public long   getSubmitTime(){ return _submitTime ; }
-   public String getStatus(){ return _status  ;}
-   public long   getJobId(){ return _jobId ; }
-
-   public String toString(){
-      StringBuilder sb = new StringBuilder();
-      sb.append(_jobId).append(';');
-      sb.append(_client).append(':').append(_clientId) ;
-      synchronized (__format) {
-          sb.append(';').append(__format.format(new Date(_startTime))).
-                  append(';').append(__format.format(new Date(_submitTime))).
-                  append(';').append(_status).append(';') ;
-      }
-      return sb.toString();
-   }
+    private final String _client;
+    private final long _clientId;
+    private final long _submitTime;
+    private final long _startTime;
+    private String _status;
+    private final long _jobId;
+
+
+    public JobInfo(long submitTime, long startTime, String status, int id, String clientName, long clientId) {
+        _submitTime = submitTime;
+        _startTime = startTime;
+        _status = checkNotNull(status);
+        _jobId = id;
+        _client = clientName;
+        _clientId = clientId;
+    }
+
+    public String getClientName() {
+        return _client;
+    }
+
+    public long getClientId() {
+        return _clientId;
+    }
+
+    public long getStartTime() {
+        return _startTime;
+    }
+
+    public long getSubmitTime() {
+        return _submitTime;
+    }
+
+    public String getStatus() {
+        return _status;
+    }
+
+    public long getJobId() {
+        return _jobId;
+    }
+
+    public String toString() {
+        StringBuilder sb = new StringBuilder();
+        sb.append(_jobId).append(';');
+        sb.append(_client).append(':').append(_clientId);
+        sb.append(';').append(LocalDateTime.ofInstant(Instant.ofEpochMilli(_submitTime), ZoneId.systemDefault()).format(__formatter));
```

Comment:
Would something like `__formatter.format(Instant.ofEpochMilli(_submitTime))` also work?  That seems less convoluted.